recipes/skidl/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "skidl" %}
+{% set version = "0.0.26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0853614f8271ae429e715cfb7a1e505af40d1e8caee1b5cf08c6d269e69e631
+
+build:
+  number: 0
```

Comment:
Do you know if this recipe cannot be noarch for the same reason as the another one?

Reply:
When I did, the windows build complained that there was an exe

Reply:
only linux builds matter for noarch

Reply:
I see. Will fix